client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/style.scss

```diff
@@ -1,22 +1,109 @@
+@import "@automattic/onboarding/styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 .trial-plan--container {
 	max-width: 692px;
-	text-align: center;
 	margin: auto;
+	padding: 1rem;
+	text-align: center;
 
 	.onboarding-title {
 		margin-bottom: 0.5rem;
 	}
 
 	.onboarding-subtitle {
-		margin-bottom: 2rem;
+		margin-bottom: 2.5rem;
+	}
+
+	.action-buttons__next {
+		padding-left: 1.5rem;
+		padding-right: 1.5rem;
 	}
 
 	p {
 		color: var(--studio-gray-100);
+		margin-bottom: 2.5rem;
 
 		a {
 			color: inherit;
 			text-decoration: underline;
 		}
 	}
+
+	.trial-plan--details {
+		display: flex;
+		text-align: initial;
+		flex-direction: column;
+		border: solid var(--studio-gray-5) 1px;
+		border-radius: 4px;
+		padding: 1.75em;
```

Comment:
This change was not part of this PR, since I branched out from the **add/migration-trial** branch. Now it's rebased with the most recent version of trunk.

Anyway, you are right; there is not reason to be `em`. I'll replace it with a `rem` unit. It was copy/paste from one existing part of the CSS.